scripts/build_anaconda.sh

```diff
@@ -117,9 +117,8 @@ fi
 if [[ -n $CONDA_INSTALL_LOCALLY ]]; then
   install_locally=1
 fi
-if [[ -n $INTEGRATED ]]; then
```

Comment:
This will turn all Caffe2-only conda packages into Caffe2+Pytorch conda packages. I think this test can just stay.